iothub/service/tests/Exceptions/IotHubServiceExceptionTests.cs

```diff
@@ -76,42 +75,6 @@ public void IotHubServiceException_Ctor_WithTransientErrorCode_Ok(IotHubServiceE
             exception.ErrorCode.Should().Be(errorCode);
         }
 
-        [TestMethod]
-        public void IotHubServiceException_UntrackedErrorCode_NotHttpStatusCodeRequestTimeout_IsNotTransient()
-        {
-            // arrange - act
-            HttpStatusCode statusCode = HttpStatusCode.MovedPermanently; // not HttpStatusCode.RequestTimeout
-
-            var exception = new IotHubServiceException(
-                Message,
-                statusCode,
-                IotHubServiceErrorCode.Unknown);
-
-            // assert
-            exception.StatusCode.Should().Be(statusCode);
-            exception.Message.Should().Be(Message);
-            exception.IsTransient.Should().BeFalse();
-            exception.TrackingId.Should().BeNull();
-        }
-
-        [TestMethod]
-        public void IotHubServiceException_UntrackedErrorCode_HttpStatusCodeRequestTimeout_IsTransient()
-        {
-            // arrange - act
-            HttpStatusCode statusCode = HttpStatusCode.RequestTimeout;
-
-            var exception = new IotHubServiceException(
-                Message,
-                statusCode,
-                IotHubServiceErrorCode.Unknown);
-
-            // assert
-            exception.StatusCode.Should().Be(statusCode);
-            exception.Message.Should().Be(Message);
-            exception.IsTransient.Should().BeTrue();
-            exception.TrackingId.Should().BeNull();
-        }
```

Comment:
We don't these 2 tests as we won't have a transient "IotHubServiceException" with "RequestTimeout" as `StatusCode` and "Unknown" as `ErrorCode`.